x-pack/plugins/infra/server/lib/alerting/inventory_metric_threshold/evaluate_condition.ts

```diff
@@ -122,14 +122,14 @@ const getData = async (
     if (!nodes.length) return { [UNGROUPED_FACTORY_KEY]: null }; // No Data state
 
     return nodes.reduce((acc, n) => {
-      const nodePathItem = last(n.path) as any;
+      const { name: nodeName } = n;
       const m = first(n.metrics);
       if (m && m.value && m.timeseries) {
         const { timeseries } = m;
         const values = timeseries.rows.map((row) => row.metric_0) as Array<number | null>;
-        acc[nodePathItem.label] = values;
+        acc[nodeName] = values;
       } else {
-        acc[nodePathItem.label] = m && m.value;
+        acc[name] = m && m.value;
```

Comment:
You probably mean

```suggestion
        acc[nodeName] = m && m.value;
```

, right?

I wondered why that passes the type check - turns out there's `window.name` which it falls back to. :scream:

Reply:
Yikes, thanks for catching that